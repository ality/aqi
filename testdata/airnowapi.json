[{"issueDate":"08/13/21","validDate":"08/13/21","recordSequence":0,"timezone":"PDT","time":"17:00","dataType":"O","isPrimary":true,"reportingArea":"Sacramento","stateCode":"CA","latitude":38.567,"longitude":-121.467,"parameter":"PM2.5","aqi":50,"category":"Good","isActionDay":false,"discussion":"","reportingAgency":"Sacramento Metro. AQMD"},{"issueDate":"08/13/21","validDate":"08/13/21","recordSequence":0,"timezone":"PDT","time":"17:00","dataType":"O","isPrimary":false,"reportingArea":"Sacramento","stateCode":"CA","latitude":38.567,"longitude":-121.467,"parameter":"OZONE","aqi":36,"category":"Good","isActionDay":false,"discussion":"","reportingAgency":"Sacramento Metro. AQMD"},{"issueDate":"08/13/21","validDate":"08/13/21","recordSequence":0,"timezone":"PDT","time":"17:00","dataType":"O","isPrimary":false,"reportingArea":"Sacramento","stateCode":"CA","latitude":38.567,"longitude":-121.467,"parameter":"PM10","aqi":31,"category":"Good","isActionDay":false,"discussion":"","reportingAgency":"Sacramento Metro. AQMD"},{"issueDate":"08/13/21","validDate":"08/13/21","recordSequence":0,"timezone":"PDT","time":"","dataType":"F","isPrimary":true,"reportingArea":"Sacramento","stateCode":"CA","latitude":38.567,"longitude":-121.467,"parameter":"OZONE","aqi":67,"category":"Moderate","isActionDay":false,"discussion":"Friday through Sunday, periods of light northwesterly winds aloft will gradually transport a dense plume of smoke over the Sacramento region. Daytime heating will increase vertical mixing, bringing the aloft smoke to the surface and increasing particle concentrations each day. However, this dense plume of smoke will hinder ozone development, despite afternoon temperatures around 100F. As a result, particle levels will be high-Moderate Friday and Unhealthy for Sensitive Groups on Saturday and Sunday. Ozone levels will be Moderate on all three days.","reportingAgency":"Sacramento Metro. AQMD"},{"issueDate":"08/13/21","validDate":"08/13/21","recordSequence":0,"timezone":"PDT","time":"","dataType":"F","isPrimary":false,"reportingArea":"Sacramento","stateCode":"CA","latitude":38.567,"longitude":-121.467,"parameter":"PM2.5","aqi":61,"category":"Moderate","isActionDay":false,"discussion":"Friday through Sunday, periods of light northwesterly winds aloft will gradually transport a dense plume of smoke over the Sacramento region. Daytime heating will increase vertical mixing, bringing the aloft smoke to the surface and increasing particle concentrations each day. However, this dense plume of smoke will hinder ozone development, despite afternoon temperatures around 100F. As a result, particle levels will be high-Moderate Friday and Unhealthy for Sensitive Groups on Saturday and Sunday. Ozone levels will be Moderate on all three days.","reportingAgency":"Sacramento Metro. AQMD"},{"issueDate":"08/13/21","validDate":"08/14/21","recordSequence":1,"timezone":"PDT","time":"","dataType":"F","isPrimary":true,"reportingArea":"Sacramento","stateCode":"CA","latitude":38.567,"longitude":-121.467,"parameter":"PM2.5","aqi":89,"category":"Moderate","isActionDay":false,"discussion":"Friday through Sunday, periods of light northwesterly winds aloft will gradually transport a dense plume of smoke over the Sacramento region. Daytime heating will increase vertical mixing, bringing the aloft smoke to the surface and increasing particle concentrations each day. However, this dense plume of smoke will hinder ozone development, despite afternoon temperatures around 100F. As a result, particle levels will be high-Moderate Friday and Unhealthy for Sensitive Groups on Saturday and Sunday. Ozone levels will be Moderate on all three days.","reportingAgency":"Sacramento Metro. AQMD"},{"issueDate":"08/13/21","validDate":"08/14/21","recordSequence":1,"timezone":"PDT","time":"","dataType":"F","isPrimary":false,"reportingArea":"Sacramento","stateCode":"CA","latitude":38.567,"longitude":-121.467,"parameter":"OZONE","aqi":61,"category":"Moderate","isActionDay":false,"discussion":"Friday through Sunday, periods of light northwesterly winds aloft will gradually transport a dense plume of smoke over the Sacramento region. Daytime heating will increase vertical mixing, bringing the aloft smoke to the surface and increasing particle concentrations each day. However, this dense plume of smoke will hinder ozone development, despite afternoon temperatures around 100F. As a result, particle levels will be high-Moderate Friday and Unhealthy for Sensitive Groups on Saturday and Sunday. Ozone levels will be Moderate on all three days.","reportingAgency":"Sacramento Metro. AQMD"},{"issueDate":"08/13/21","validDate":"08/15/21","recordSequence":2,"timezone":"PDT","time":"","dataType":"F","isPrimary":true,"reportingArea":"Sacramento","stateCode":"CA","latitude":38.567,"longitude":-121.467,"parameter":"PM2.5","aqi":99,"category":"Moderate","isActionDay":false,"discussion":"Friday through Sunday, periods of light northwesterly winds aloft will gradually transport a dense plume of smoke over the Sacramento region. Daytime heating will increase vertical mixing, bringing the aloft smoke to the surface and increasing particle concentrations each day. However, this dense plume of smoke will hinder ozone development, despite afternoon temperatures around 100F. As a result, particle levels will be high-Moderate Friday and Unhealthy for Sensitive Groups on Saturday and Sunday. Ozone levels will be Moderate on all three days.","reportingAgency":"Sacramento Metro. AQMD"},{"issueDate":"08/13/21","validDate":"08/15/21","recordSequence":2,"timezone":"PDT","time":"","dataType":"F","isPrimary":false,"reportingArea":"Sacramento","stateCode":"CA","latitude":38.567,"longitude":-121.467,"parameter":"OZONE","aqi":51,"category":"Moderate","isActionDay":false,"discussion":"Friday through Sunday, periods of light northwesterly winds aloft will gradually transport a dense plume of smoke over the Sacramento region. Daytime heating will increase vertical mixing, bringing the aloft smoke to the surface and increasing particle concentrations each day. However, this dense plume of smoke will hinder ozone development, despite afternoon temperatures around 100F. As a result, particle levels will be high-Moderate Friday and Unhealthy for Sensitive Groups on Saturday and Sunday. Ozone levels will be Moderate on all three days.","reportingAgency":"Sacramento Metro. AQMD"},{"issueDate":"08/13/21","validDate":"08/16/21","recordSequence":3,"timezone":"PDT","time":"","dataType":"F","isPrimary":true,"reportingArea":"Sacramento","stateCode":"CA","latitude":38.567,"longitude":-121.467,"parameter":"OZONE","aqi":74,"category":"Moderate","isActionDay":false,"discussion":"Friday through Sunday, periods of light northwesterly winds aloft will gradually transport a dense plume of smoke over the Sacramento region. Daytime heating will increase vertical mixing, bringing the aloft smoke to the surface and increasing particle concentrations each day. However, this dense plume of smoke will hinder ozone development, despite afternoon temperatures around 100F. As a result, particle levels will be high-Moderate Friday and Unhealthy for Sensitive Groups on Saturday and Sunday. Ozone levels will be Moderate on all three days.","reportingAgency":"Sacramento Metro. AQMD"},{"issueDate":"08/13/21","validDate":"08/16/21","recordSequence":3,"timezone":"PDT","time":"","dataType":"F","isPrimary":false,"reportingArea":"Sacramento","stateCode":"CA","latitude":38.567,"longitude":-121.467,"parameter":"PM2.5","aqi":68,"category":"Moderate","isActionDay":false,"discussion":"Friday through Sunday, periods of light northwesterly winds aloft will gradually transport a dense plume of smoke over the Sacramento region. Daytime heating will increase vertical mixing, bringing the aloft smoke to the surface and increasing particle concentrations each day. However, this dense plume of smoke will hinder ozone development, despite afternoon temperatures around 100F. As a result, particle levels will be high-Moderate Friday and Unhealthy for Sensitive Groups on Saturday and Sunday. Ozone levels will be Moderate on all three days.","reportingAgency":"Sacramento Metro. AQMD"},{"issueDate":"08/13/21","validDate":"08/17/21","recordSequence":4,"timezone":"PDT","time":"","dataType":"F","isPrimary":true,"reportingArea":"Sacramento","stateCode":"CA","latitude":38.567,"longitude":-121.467,"parameter":"OZONE","aqi":54,"category":"Moderate","isActionDay":false,"discussion":"Friday through Sunday, periods of light northwesterly winds aloft will gradually transport a dense plume of smoke over the Sacramento region. Daytime heating will increase vertical mixing, bringing the aloft smoke to the surface and increasing particle concentrations each day. However, this dense plume of smoke will hinder ozone development, despite afternoon temperatures around 100F. As a result, particle levels will be high-Moderate Friday and Unhealthy for Sensitive Groups on Saturday and Sunday. Ozone levels will be Moderate on all three days.","reportingAgency":"Sacramento Metro. AQMD"},{"issueDate":"08/13/21","validDate":"08/17/21","recordSequence":4,"timezone":"PDT","time":"","dataType":"F","isPrimary":false,"reportingArea":"Sacramento","stateCode":"CA","latitude":38.567,"longitude":-121.467,"parameter":"PM2.5","aqi":42,"category":"Good","isActionDay":false,"discussion":"Friday through Sunday, periods of light northwesterly winds aloft will gradually transport a dense plume of smoke over the Sacramento region. Daytime heating will increase vertical mixing, bringing the aloft smoke to the surface and increasing particle concentrations each day. However, this dense plume of smoke will hinder ozone development, despite afternoon temperatures around 100F. As a result, particle levels will be high-Moderate Friday and Unhealthy for Sensitive Groups on Saturday and Sunday. Ozone levels will be Moderate on all three days.","reportingAgency":"Sacramento Metro. AQMD"},{"issueDate":"08/13/21","validDate":"08/18/21","recordSequence":5,"timezone":"PDT","time":"","dataType":"F","isPrimary":true,"reportingArea":"Sacramento","stateCode":"CA","latitude":38.567,"longitude":-121.467,"parameter":"OZONE","aqi":51,"category":"Moderate","isActionDay":false,"discussion":"Friday through Sunday, periods of light northwesterly winds aloft will gradually transport a dense plume of smoke over the Sacramento region. Daytime heating will increase vertical mixing, bringing the aloft smoke to the surface and increasing particle concentrations each day. However, this dense plume of smoke will hinder ozone development, despite afternoon temperatures around 100F. As a result, particle levels will be high-Moderate Friday and Unhealthy for Sensitive Groups on Saturday and Sunday. Ozone levels will be Moderate on all three days.","reportingAgency":"Sacramento Metro. AQMD"},{"issueDate":"08/13/21","validDate":"08/18/21","recordSequence":5,"timezone":"PDT","time":"","dataType":"F","isPrimary":false,"reportingArea":"Sacramento","stateCode":"CA","latitude":38.567,"longitude":-121.467,"parameter":"PM2.5","aqi":33,"category":"Good","isActionDay":false,"discussion":"Friday through Sunday, periods of light northwesterly winds aloft will gradually transport a dense plume of smoke over the Sacramento region. Daytime heating will increase vertical mixing, bringing the aloft smoke to the surface and increasing particle concentrations each day. However, this dense plume of smoke will hinder ozone development, despite afternoon temperatures around 100F. As a result, particle levels will be high-Moderate Friday and Unhealthy for Sensitive Groups on Saturday and Sunday. Ozone levels will be Moderate on all three days.","reportingAgency":"Sacramento Metro. AQMD"}]