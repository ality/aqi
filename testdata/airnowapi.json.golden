Friday through Sunday, periods of light northwesterly winds aloft will
gradually transport a dense plume of smoke over the Sacramento region.
Daytime heating will increase vertical mixing, bringing the aloft
smoke to the surface and increasing particle concentrations each day.
However, this dense plume of smoke will hinder ozone development,
despite afternoon temperatures around 100F. As a result, particle
levels will be high-Moderate Friday and Unhealthy for Sensitive Groups
on Saturday and Sunday. Ozone levels will be Moderate on all three
days.
